backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -108,38 +105,38 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     .as[Option[String]]("temporary-directory").getOrElse("""$(mktemp -d "$PWD"/tmp.XXXXXX)""")
 
   /**
-    * Maps WdlFile objects for use in the commandLinePreProcessor.
+    * Maps WomFile objects for use in the commandLinePreProcessor.
     *
-    * By default just calls the pass through mapper mapCommandLineWdlFile.
+    * By default just calls the pass through mapper mapCommandLineWomFile.
     *
     * Sometimes a preprocessor may need to localize the files, etc.
     *
-    * @param wdlFile The wdlFile.
-    * @return The updated wdlFile.
+    * @param womFile The womFile.
+    * @return The updated womFile.
     */
-  def preProcessWdlFile(wdlFile: WomFile): WomFile = wdlFile
+  def preProcessWomFile(womFile: WomFile): WomFile = womFile
 
   /** @see [[Command.instantiate]] */
   final lazy val commandLinePreProcessor: WomEvaluatedCallInputs => Try[WomEvaluatedCallInputs] = {
     inputs =>
-      TryUtil.sequenceMap(inputs mapValues WomFileMapper.mapWomFiles(preProcessWdlFile)).
+      TryUtil.sequenceMap(inputs mapValues WomFileMapper.mapWomFiles(preProcessWomFile)).
         recoverWith {
           case e => Failure(new IOException(e.getMessage) with CromwellFatalExceptionMarker)
         }
   }
 
   /**
-    * Maps WdlFile to a local path, for use in the commandLineValueMapper.
+    * Maps WomFile to a local path, for use in the commandLineValueMapper.
     *
-    * @param wdlFile The wdlFile.
-    * @return The updated wdlFile.
+    * @param womFile The womFile.
+    * @return The updated womFile.
```

Comment:
ToL I always hate these "that's the name of the parameter!" parrot-docs

Reply:
Agreed. Unless the scaladoc is telling me something non-obvious from the name/type I see no point in having it

Reply:
I'll whack this in the sequel PR.